src/error/__tests__/GraphQLError-test.ts

```diff
@@ -12,6 +10,10 @@ import { parse } from '../../language/parser';
 import { Source } from '../../language/source';
 
 import { GraphQLError, printError } from '../GraphQLError';
+import {
+  OperationDefinitionNode,
+  ObjectTypeDefinitionNode,
+} from '../../language/ast';
```

Comment:
Please use type imports for all new imported types also please try to match order of imports.

Reply:
Please review your PR and apply it to all other places.

Reply:
Fixed the type imports. What do you mean by order? Do we have a lint rule for that? (otherwise, it will be difficult to enforce / fix)